libs/bower/README.md

```diff
@@ -0,0 +1,25 @@
+# How to add dependencies with bower in ILIAS
+
+**New dependencies need to be approved by the Jour Fixe of the ILIAS society or on behalf of 
+the Technical Board.**
+
+## Dependencies for production
+- Comment all lines in libs/.gitignore which begin with bower/
+- Add a new library using bower, e.g. "bower install bootstrap@3.3.7 --save"
+- Add a section in "extra" with the following metadata":
+```json
+ "jquery": {
```

Comment:
I think we should streamline this with the according info in the composer.json sometime.